x-pack/legacy/plugins/infra/public/graphql/types.ts

```diff
@@ -291,7 +291,7 @@ export interface InfraSnapshotNodePath {
 }
 
 export interface InfraSnapshotNodeMetric {
-  name: InfraSnapshotMetricType;
+  name: 'count' | 'cpu' | 'load' | 'memory' | 'tx' | 'rx' | 'logRate';
```

Comment:
This should be `SnapshotMetricType`.

Reply:
Looks like we already have a type for this in `common/inventory_models/types.ts`